src/js/state/View/types.js

```diff
@@ -1,55 +1,21 @@
 /* @flow */
 
-export type InvestigationView = "tree" | "linear"
 export type ViewState = {
```

Comment:
We don't need a migration for this since we are deleting and never reading from them again. The next time the state is saved, they will be removed.